src/App.js

```diff
@@ -2,18 +2,30 @@ import React, { Component } from "react";
 
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
+import Heading from "./Heading"
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer"
 import "./App.css";
+import "./grid.css";
 
 class App extends Component {
   render() {
-    return (
-      <div className="App">
-        <header className="App-header">CYF Hotel</header>
+    const contactInfo = [
+      "123 Fake Street, London, E1 4UD",
+      "hello@fakehotel.com",
+      "0123 456789"
+    ];
+   return (
+    <div className="App">
+      <div className="container">
+        <Heading />
+        <TouristInfoCards />
         <Bookings />
-        <Restaurant />
+        <Restaurant text={"add"}/>
```

Comment:
This isn't necessarily incorrect, but it is a little unusual. We can't have to pass down props for every bit of data in our application - our `App` component would be very complex then!

In this case, I would say that the `App` component "doesn't care" about what text the button has inside `Restaurant`. That's the job of the `Restaurant` component. This is sort of what I meant when I was talking about how components should have 1 *responsibility*

Reply:
Looking back at my code, I realise there is a better way to do this, will be updating this line shortly